bitnami/mysql/templates/secrets.yaml

```diff
@@ -1,34 +1,39 @@
-{{- if (not .Values.existingSecret) -}}
+{{- if (include "mysql.createSecret" .) }}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ template "mysql.fullname" . }}
-  labels: {{- include "mysql.labels" . | nindent 4 }}
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
 type: Opaque
 data:
-  {{- if .Values.root.password }}
-  mysql-root-password: {{ .Values.root.password | b64enc | quote }}
-  {{- else if (not .Values.root.forcePassword) }}
+  {{- if not (empty .Values.auth.rootPassword) }}
```

Comment:
Just for curiousity, and empty string would not be evaluated to false?

Reply:
Yes, but this explicitly states it.